src/task-4.js

```diff
@@ -1,6 +1,8 @@
 
```

Comment:
I see couple of tests are failing, please fix

Reply:
Surprising( 10 of 10 tests were succesfully passed on my machine